### PR TITLE
Timesheets test

### PR DESCRIPTION
**Pull request by jkrobicki**

### CODE REVIEW

**Comment by acivitillo** [comment]
@jkrobicki congrats for your first PR into this project. Here is my feedback:

- you should remove .ipynb files from the source code folders
- review the file names, you have a few filenames like "Untitled"
- remove sqlite databases, databases should not be version controlled
- names like `main_test2.py ` are not good names, please review the file names

**Comment by jkrobicki** [comment]
@acivitillo, following your feedback I've updated my commit

File: backend/main.py [hunk 1]
```diff
@@ -1,7 +1,8 @@
 from fastapi import FastAPI
 from sqlmodel import Session, select, SQLModel
 from sqlalchemy.exc import OperationalError
-from models import Epic, TimeLog, engine, create_db
+from models import Epic, TimeLog, User, engine, create_db, Client
+from utils import string_to_datetime, initials
 import datetime
 
 # strftime("%m/%d/%Y, %H:%M:%S")
```
Comment (acivitillo) [inline backend/main.py:8]: please have a look at the code for comments that can be removed or explained. You do not need to remove comments, but you need to make sure they are easy to understand and they serve a purpose

File: backend/models.py [hunk 2]
```diff
@@ -1,22 +1,52 @@
...
+    user_initials: str
     start_time: str
     end_time: str
+    client_name: str
+    epic_name: str
+    work_hours: Optional[float]
+    month: Optional[int]
+    year: Optional[int]
+
+    # @validator('month', always=True, pre=True)
```
Comment (acivitillo) [inline backend/models.py:21]: do you need these comments?

File: backend/utils.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import datetime
+
+
+def string_to_datetime(date_string):
+    date = datetime.datetime.strptime(date_string, "%Y-%m-%dT%H:%M:%S.%fZ")
+    return date
+
+
+def initials(full_name):
```
Comment (acivitillo) [inline backend/utils.py:9]: what would be a better name for this function?

**Review by acivitillo — CHANGES_REQUESTED** [review]
in the code, review:

- comments, they need to serve a purpose
- naming conventions, in general `initials` is not a good name, initials of what? What is this function doing? Why do we need this function?

**Comment by jkrobicki** [comment]
@acivitillo, I've updated my commit and closed resolved issued

**Pull request merged**